netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java

```diff
@@ -79,14 +79,18 @@
    * noticing changes to DNS.
    */
   public static NettyChannelBuilder forAddress(SocketAddress serverAddress) {
```

Comment:
I don't know. If the answer is yes, we shouldn't make this change. If the answer is no, they should have read the javadocs and use `forTarget()` and we wouldn't need to make this change. So not making the change sounds safer to me.

The problem is inherent in InetSocketAddress. People are already hit by it [elsewhere](https://www.tellapart.com/inetsocketaddress-considered-harmful/). I don't think gRPC should compensate for this.